ntoskrnl/mm/ARM3/mminit.c

```diff
@@ -2118,6 +2118,7 @@ MmArmInitSystem(IN ULONG Phase,
         TestPte = MiProtoPteToPte(&TempPte);
         ASSERT(PointerPte == TestPte);
 
+#ifndef _M_AMD64 // Not working on x64 for obvoius reason
```

Comment:
Can't we change this for if(_M_X86_) or whatever x86-32bits-only relevant define is ?